Cargo.toml

```diff
@@ -29,9 +29,6 @@ path = "src/bin/cargo_kani.rs"
 [profile.release]
 strip = "debuginfo"
 
-# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I don't know why we had two copyright statements in this file. @tedinski?

Reply:
lol, this was probably a copy and paste I never cleaned up.